dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/ScanContext.java

```diff
@@ -79,11 +90,20 @@ void endWalkDir(FileStatus dir, long nFiles, long nDirs, long accumFileSize) thr
       this.numDirs += nDirs;
       this.accumulatedFileSize += accumFileSize;
 
+      String absolutePath = dir.getPath().toUri().getPath();
+      HdfsFileStatus hdfsFileStatus = dfsClient.getFileInfo(absolutePath);
+      byte byteStoragePolicy = hdfsFileStatus.getStoragePolicy();
+      StoragePolicy storagePolicy = StoragePolicy.valueOf(byteStoragePolicy);
+
       csvPrinter.printRecord(
-          dir.getPath().toUri().getPath(),
+          absolutePath,
           dir.getOwner(),
           dir.getGroup(),
-          dir.getPermission().toString());
+          dir.getPermission(),
+          dir.getModificationTime(),
```

Comment:
I would suggest converting the modification time from the unix epoch milli format to ISO, e.g. `2024-06-21 12:34:56.123`.

See for example:
https://github.com/google/dwh-migration-tools/blob/7399219ad87ccfd15bf16e2eae702e5ba00c4cff/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterUsageMetricsTask.java#L96